src/components/Footer.jsx

```diff
@@ -1,11 +1,18 @@
 const Footer = () => {
   return (
     <div className="text-center bg-[#daf0ff] m-5">
-      <p className="mt-4 text-yellow-400 font-bold text-2xl inline-block"> UCR</p>
-      <p className="inline-block ml-2 font-bold text-2xl text-gray-700">Senior Designs</p>
-      <p className="text-xs mb-12">Copyright © ACM UCR - All Rights Reserved.</p>
+      <p className="mt-4 text-yellow-400 font-bold text-2xl inline-block">
+        {" "}
+        UCR
+      </p>
+      <p className="inline-block ml-2 font-bold text-2xl text-gray-700">
```

Comment:
- don't need inline-block
- the Senior Design text isn't gray, you're not limited to the color's I've provided, use an RGB color finder to get the correct text color from the figma file